### DISCUSSION TITLE
making a pwa

### DISCUSSION THREAD

**Discussion by BobStadler** [copy/v86#1150] [Ideas]
hi, i have a suggestion.

would it be possible to make this a pwa so it is installable as an web app and one can install it for offline use?

best regards

**Comment by SuperMaxusa** [comment]
See #194 and #975, as far as I understand, for PWA we can use Service Workers and Cache API for offline usage (ie. load `libv86.js`, `v86.wasm`, bios and images): https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Tutorials/js13kGames/Offline_Service_workers

**Reply by BobStadler** [reply]
so is this planned?
